java/buffer/build.gradle

```diff
@@ -17,8 +17,14 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    compileOptions {
+        targetCompatibility rootProject.ext.targetJavaVersion
+        sourceCompatibility rootProject.ext.sourceJavaVersion
+    }
 }
 
 dependencies {
     // arcgis-android & appcompat lib dependencies from rootProject build.gradle
+    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
```

Comment:
I think you might want to use this instead 
`implementation "com.android.support.constraint:constraint-layout:$constraintLayoutVersion"`

Also, our constraint layout version in the version.gradle file is 1.1.0, should i create an issue to change it to 1.1.2 ?